src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -318,6 +324,96 @@ public override IRowCursor[] GetRowCursorSet(out IRowCursorConsolidator consolid
             return cursors;
         }
 
+        public void SaveAsOnnx(OnnxContext ctx)
+        {
+            if (_mapper is ISaveAsOnnx onnx)
+                onnx.SaveAsOnnx(ctx);
+        }
+
+        public void SaveAsPfa(BoundPfaContext ctx)
+        {
+            if (_mapper is ISaveAsPfa onnx)
+                onnx.SaveAsPfa(ctx);
+        }
+
+        public Func<int, bool> GetDependencies(Func<int, bool> predicate)
+        {
+            Func<int, bool> predicateInput;
+            var active = _bindings.GetActive(predicate, out predicateInput);
+            return predicateInput;
+        }
+
+        public IRow GetRow(IRow input, Func<int, bool> active, out Action disposer)
+        {
+            Host.CheckValue(input, nameof(input));
+            Host.CheckValue(active, nameof(active));
+            Host.Check(input.Schema == Source.Schema, "Schema of input row must be the same as the schema the mapper is bound to");
+
+            disposer = null;
+            using (var ch = Host.Start("GetEntireRow"))
+            {
+                Action disp;
+                var activeArr = new bool[Schema.ColumnCount];
+                for (int i = 0; i < Schema.ColumnCount; i++)
+                    activeArr[i] = active(i);
+                var pred = _bindings.GetActiveOutputColumns(activeArr);
+                var getters = _mapper.CreateGetters(input, pred, out disp);
+                disposer += disp;
+                ch.Done();
+                return new Row(input, this, Schema, getters);
+            }
+        }
+
+        private sealed class Row : IRow
```

Comment:
>Row [](start = 29, length = 3)

Can I use SimpleRow?

Reply:
I suppose you can

---
In reply to: [214102719](https://github.com/dotnet/machinelearning/pull/759#discussion_r214102719) [](ancestors = 214102719)

Reply:
A better question is: can you reuse `RowCursor` somehow?

---
In reply to: [214105932](https://github.com/dotnet/machinelearning/pull/759#discussion_r214105932) [](ancestors = 214105932,214102719)

Reply:
It's rude to intervene my dialogue with myself!

---
In reply to: [214106201](https://github.com/dotnet/machinelearning/pull/759#discussion_r214106201) [](ancestors = 214106201,214105932,214102719)